data/manishd17.json

```diff
@@ -0,0 +1,48 @@
+{
+	"name": "Manish Jha",
+	"displayStatsPublic": true,
+	"type": "personal",
```

Comment:
We no longer need the `displayStatsPublic`, `type` keys. Please remove them from your `json`. 


```suggestion
```

Reply:
Ok, Thank you